app/src/androidTest/java/com/android/sample/screen/CoachCaptureScreenTest.kt

```diff
@@ -0,0 +1,65 @@
+package com.android.sample.screen
+
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.test.assertHasClickAction
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import com.android.sample.model.camera.CameraViewModel
+import com.android.sample.ui.mlFeedback.CoachCaptureScreen
+import com.android.sample.ui.navigation.NavigationActions
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.Mockito.mock
+
+class CoachCaptureScreenTest {
+  private lateinit var mockCameraViewModel: CameraViewModel
```

Comment:
Why mock the CameraViewModel if not to use it ?

Reply:
Good question. Changed.